cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Handle.java

```diff
@@ -769,6 +772,13 @@ private int runStatementInt(String sql, List<Integer> types) {
         sql = sql.trim();
         String lowerSql = sql.toLowerCase();
 
```

Comment:
Changed this to mimic what we do in cdb2api.c: every statement we consume everything from the database and clear responses before processing set directives .. this alone might be sufficient to fix cdb2jdbc (I'm testing that now).